test/suite/commands/seek.md

```diff
@@ -250,3 +250,50 @@ ghi
 jkl
 mno
 ```
+
+# 4
```

Comment:
Could you add a test when seeking backward? I don't know what the expected result is, but there might be some problems with the selection not extending exactly as expected.